src/pages/ProductPage.jsx

```diff
@@ -52,83 +53,101 @@ const ProductPage = () => {
+          <div className={styles.IconButton}>
+            <IconButton
+              faIcon={faArrowUpFromBracket}
+              label="Share"
+              onClick={handleShare}
+            />
+            <IconButton faIcon={faHeart} label="Save" onClick={handleSave} />
+          </div>
+        </div>
+        <ProductGallery
+          bigImage={
+            "https://a0.muscache.com/im/pictures/hosting/Hosting-U3RheVN1cHBseUxpc3Rpbmc6MTEzNDc1NzYxMjc3MDc0NzgxMg%3D%3D/original/5ad7780d-76b5-428f-9219-432243a83a03.jpeg"
+          }
+          smallTopLeftImage={
+            "https://a0.muscache.com/im/pictures/hosting/Hosting-U3RheVN1cHBseUxpc3Rpbmc6MTEzNDc1NzYxMjc3MDc0NzgxMg%3D%3D/original/253a0690-2a1e-4c34-ae7f-968b869be4b5.jpeg"
+          }
+          smallTopRightImage={
+            "https://a0.muscache.com/im/pictures/hosting/Hosting-U3RheVN1cHBseUxpc3Rpbmc6MTEzNDc1NzYxMjc3MDc0NzgxMg%3D%3D/original/78ed3027-a197-4043-9b7e-8fc79a5425fc.jpeg"
+          }
+          smallBottomLeftImage={
+            "https://a0.muscache.com/im/pictures/hosting/Hosting-U3RheVN1cHBseUxpc3Rpbmc6MTEzNDc1NzYxMjc3MDc0NzgxMg%3D%3D/original/182abee3-f7f8-4652-8c2a-f845e990d9c5.jpeg"
+          }
+          smallBottomRightImage={
+            "https://a0.muscache.com/im/pictures/hosting/Hosting-U3RheVN1cHBseUxpc3Rpbmc6MTEzNDc1NzYxMjc3MDc0NzgxMg%3D%3D/original/2ca9c23e-85db-48f8-bd21-0718c286dcdf.jpeg"
+          }
         />
-        <IconButton
-          faIcon={faArrowUpFromBracket}
-          label="Share"
-          onClick={handleShare}
+        <div className={styles.ProductDescriptionContainer}>
+          <div className={styles.ProductDescription}>
+            <ProductSummary
+              accommodation={"Entire rental unit"}
+              address={"Berlin, Germany"}
+              guests={{ key: "guest", value: 3 }}
+              bedrooms={{ key: "bedroom", value: 1 }}
+              beds={{ key: "bed", value: 3 }}
+              baths={{ key: "bath", value: 1 }}
+              starGrade={4.65}
+              reviews={23}
+            />
+
+            <FavoriteStay />
+            <HostSummary
+              hostName="Raus"
+              hostingDuration={1}
+              role="Superhost"
+              profilePicUrl={hostImage}
+            />
+            <ProductHighlight highlights={highlights} />
+            <hr />
+            <ProductDescription
+              descriptionPlace="The apartment consists of a large living room, a private, large bathroom with a bathtub and a high space, which is suitable for the storage of luggage and is accessed by a staircase.accessed by a staircase"
+              descriptionSpace="The apartment starts from a quiet courtyard and has its own entrance, which does not depart from the general stairwell."
+              guestAccess="You have access to all areas of the flat. The flat has its own entrance."
+              otherThings="Do not smoke in rooms!"
+            />
+          </div>
```

Comment:
It would be great to extract this part in a separate variable as well.

Reply:
It is related to another ticket.